### PR TITLE
124 link urls in data grid to website

### LINKED ISSUE #124: Link urls in data grid to websites
The user should be able to click website and social media links. The links should open a new page in another tab.

Screen Shot 2021-09-20 at 7 56 35 PM

### PR DESCRIPTION
**Pull request by iatenine**
Website and social media links now functional. Open in new tabs. Closes #124

### CODE REVIEW

File: frontend/src/components/LeadRow.js [hunk 1]
```diff
@@ -1,22 +1,34 @@
...
 
+const SocialMediaLink = ({ lead }) => {
+  const link =
+    lead['facebook'] ||
+    lead['linkedin'] ||
+    lead['twitter'] ||
+    lead['instagram'] ||
+    null;
+
+  return <a href={link}>{link}</a>;
```
Comment (galbwe) [inline frontend/src/components/LeadRow.js:15]: Can you make this open in a new tab as well?

**Review by iatenine — COMMENTED** [review]
Updated to open in new tab

File: frontend/src/components/LeadRow.js (hunk 1 above)
Comment (iatenine) [reply frontend/src/components/LeadRow.js]: Done!

**Comment by galbwe** [comment]
@iatenine looks good. 

Can you run the formatting and linting scripts so the `lint-frontend` action passes? To do that `cd` to the `frontend` directory and run

```bash
yarn format
```

and then 

```bash
yarn lint
```

and make the requested changes until the linter passes. Then make a commit with the formatting changes.

**Review by galbwe — APPROVED** [review]
Looks good. Thanks for taking care of the formatting thing.

**Pull request merged**